grisera_api/graph_api_service.py

```diff
@@ -53,10 +53,37 @@ def create_properties(self, node_id: int, node_model: BaseModel):
         """
         node_dict = node_model.dict()
         request_body = [{"key": key, "value": value} for key, value in node_dict.items()
-                        if value is not None and key != 'additional_properties']
+                        if value is not None and key != 'additional_properties' and key != 'authors' and key != 'publication']
```

Comment:
I think `.. and key not in ['additional_properties', 'authors', 'publication']` would be more clear